src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -276,7 +276,7 @@ public function iSetItsMetaDescriptionToTooLongStringIn(string $localeCode): voi
     public function iSetItsNonTranslatableAttributeTo(ProductAttributeInterface $attribute, string $value): void
     {
         $this->client->addSubResourceData(
-        'attributes',
+            'attributes',
```

Comment:
Note for the future, don't run CS fixing on unrelated files, it just bloats the PR and there is an automated process taking care of it